docs/decisions/0008-lti-feature-separation.rst

```diff
@@ -0,0 +1,48 @@
+0008 LTI Feature Control
+#########################################################
+
+Status
+******
+**Proposed**
+
+Context
+*******
+
+Even after the refactoring described in :doc:`0006-pluggable-lti-configuration.rst` and :doc:`0007-lti-1p3-decoupling.rst` the xblock-lti-consumer library has features which will not work outside of the LMS.
+
+For example, grading endpoints modify objects which exist only in the LMS. NRPS requires access to the full user object. If you attempt to use these LTI features in another IDA such as exams, they will fail.
+
+The :code:`LtiConfiguration` model contains data for all of these features even though not all of them will be used and may not even be available in non-LMS installations.
+
+On the plus side, the first planned LTI use outside of the LMS is in the exam service where it will be managed by mode knowledgeable site staff via the admin pages rather than by course staff. And LTI use outside of an xblock requires code support, so even before this setup it will have to work in the IDA context during development.
+
+What if anything should we do to prevent LTI from being configured in impossible ways?
```

Comment:
#NIT this sentence is hard to understand for me. Could you please simplify, or make it more direct?